core/eolearn/tests/test_eotask.py

```diff
@@ -44,6 +72,20 @@ def test_chained(self):
         for i in range(5):
             self.assertEqual(composite(i), 6 * i + 9)
 
+    def test_execution_handling(self):
+        task = ExceptionTestingTask('test_exception')
+        self.assertRaises(TestException, task, 'test')
```

Comment:
At least 1 of these tests should also verify that an error message has been edited
```Python
try:
    task('test')
except TestException as exception:
    message = str(exception)
    self.assertTrue(message.startswith('During execution of task TestException: '))
```

Reply:
Ah, the upper code is not ok. But this would work:

```Python
for parameter, exception_type in [('test_exception_fail', TypeError), ('value_error', ValueError)]:
    task = ExceptionTestingTask(parameter)
    self.assertRaises(exception_type, task, 'test')
    try:
        task('test')
    except exception_type as exception:
        message = str(exception)
        self.assertTrue(message.startswith('During execution of task ExceptionTestingTask: '))
```

Reply:
Done.